traiders/backend/api/serializers/equipment.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from ..models import Equipment
+
+
+class EquipmentSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Equipment
+        fields = ["name", "category", "symbol"]
```

Comment:
also add "url" to fields. Not sure where it may be used but it is good practice to have a unique identifier for each object.